partner_group/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    group_id = fields.Many2one(
```

Comment:
Can't we use `parent_id` instead? :thinking:

Reply:
No, Jairo... jeje..